app/src/main/res/xml/preferences.xml

```diff
@@ -76,6 +76,13 @@
             android:widgetLayout="@layout/widget_image_next"
             />
 
+        <Preference
+            android:key="quests.deletion"
+            android:title="@string/pref_title_quests_deletion"
+            android:summary="@string/pref_title_quests_deletion_summary"
+            android:widgetLayout="@layout/widget_image_delete"
```

Comment:
I misuse the widget layout in some cases only to signal that something is a subpage or that something leads out of this app (into a browser). It should not be used to generally display an icon.